gem/settings/dev.py

```diff
@@ -5,7 +5,7 @@
     'localhost',
     '.localhost',
     '127.0.0.1',
-    '192.168.101.251'
+    '172.30.1.163'
 ]
```

Comment:
is this something intended for this changeset?

Reply:
@moh-moola Updated.